### ISSUE TITLE
Windows compiler error

### ISSUE THREAD

**Issue by 00110001011100010111100001111010** [radioegor146/native-obfuscator#81]
Can't compile in Windows.
```
MSBuild version 17.10.4+10fbfbf2e for .NET Framework
  1>Checking Build System
  Building Custom Rule D:/a/RatPlugin/RatPlugin/CMakeLists.txt
  native_jvm.cpp
  native_jvm_output.cpp
  string_pool.cpp
D:\a\RatPlugin\RatPlugin\string_pool.cpp(4,17): error C2466: cannot allocate an array of constant size 0 [D:\a\RatPlugin\RatPlugin\build\native_library.vcxproj]
D:\a\RatPlugin\RatPlugin\native_jvm_output.cpp(11,16): error C2466: cannot allocate an array of constant size 0 [D:\a\RatPlugin\RatPlugin\build\native_library.vcxproj]
```
native_jvm_output.cpp, 11
```
reg_method reg_methods[0];
```
string_pool.cpp, 4
```
static char pool[0LL] = {  };
```

but compiles well in linux
![chrome_9rr6J0hJUE](https://github.com/radioegor146/native-obfuscator/assets/148702857/4b67b61b-9017-4644-a560-213577940336)

**Comment by radioegor146** [comment]
Well, I believe that bug is because you don't have any classes to compile for native

**Comment by radioegor146** [comment]
Fixed in b05429f
